server/routes/index.ts

```diff
@@ -17,12 +17,15 @@ import populateCurrentUser from '../middleware/populateCurrentUser'
 import flashMessages from '../middleware/flashMessageMiddleware'
 import fromReviewMiddleware from '../middleware/fromReviewMiddleware'
 import PrisonerController from './prisonerController'
+import WhatsNewPage from './whatsNewPage/whatsNewPage'
 
 export default function Index(services: Services): Router {
   const router = Router({ mergeParams: true })
   const prisonerController = new PrisonerController(services.prisonerService)
+  const whatsNewPage = new WhatsNewPage()
 
   router.get('/prisoner/:nomsId/image', prisonerController.getImage())
```

Comment:
this should be authenticated and we should have a user in context so should sit below the rest of the middleware along with the other routes